### PR TITLE
Adds more staff-name commands to Discord

### PR DESCRIPTION
**Pull request by Partheo**
Adds:
!apples for Atomicapples
!oak for OakBoscage
!kay for TehLadyK
!kayozz for Kayozz11
!trump for AtRealDonaldTrump
!ktlwjec for Ktlwjec
!simon for Simonset55
!manatee for MenacingManatee
and finally, !partheo for me.

I've also gone through a lot of the other commands and either fixed typos, changed command descriptions, and replaced images that were no longer hosted.

### CODE REVIEW

**Comment by Atomicapples** [comment]
*facepalms excitedly*

**Comment by MenacingManatee** [comment]
No !Manatee? Where's the love man...

File: models/Discord/Commands/DiscordCommandApples.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandApples extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("apples", "Council speedrunner", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["http://i.imgur.com/AnUmGx3.jpg", "http://i.imgur.com/AGMzoP7.png", "http://i.imgur.com/aUkVKnf.png", "http://i.imgur.com/uotAwAy.png", "http://i.imgur.com/NKeMihP.png", "http://i.imgur.com/ruSaMoS.png", "http://i.imgur.com/wP89Euc.png", "http://i.imgur.com/By4rdWL.jpg", "http://i.imgur.com/XljVPoq.jpg", "http://i.imgur.com/UYExMSS.jpg", "http://i.imgur.com/7sklrgF.jpg"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandApples.js]: Please put each picture into their own line, as in !arcticfox

File: models/Discord/Commands/DiscordCommandKay.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandKay extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("kay", "Removed due to public demand", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["http://labourman.co.za/wp-content/uploads/2016/06/demotion.jpg", "https://www.matrixgroup.net/thematrixfiles/wp-content/uploads/2013/06/demotion1.jpg", "http://thenobleheart.com/wp-content/uploads/2014/03/Demotion-r1.jpg", "http://i.imgur.com/RHS5y8E.jpg", "http://i.imgur.com/oPQ9e1E.jpg", "http://corporatecrossovers.com/wp-content/uploads/2014/11/lady-demotion-300x294.jpg"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandKay.js]: Again, different lines

File: models/Discord/Commands/DiscordCommandKayozz.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandKayozz extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("kayozz", "Pictures of the only goodmin", undefined, subsystem, true);
+  
+
+  onRun(message, permissions, args) {
+    var responses = ["https://i.giphy.com/media/u0dZqvLqw9FHW/giphy.mp4", "https://i.giphy.com/media/kgyt7vWYxecfe/giphy.mp4", "http://i.imgur.com/RZJIKU9.jpg", "https://images.halloweencostumes.com/products/21768/1-1/child-mime-costume.jpg", "http://i.imgur.com/GukvQc7.png", "http://img.freepik.com/free-photo/mime-shows-at-something-invisible-on-his-palm_1304-2808.jpg?size=338&ext=jpg", "https://cdn.heavencostumes.com.au/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/s/l/sl-s3066-hush-mime-french-jester-sexy-womens-costume-close.jpg", "https://farm2.static.flickr.com/1683/25344054344_71741837ac_m.jpg", "https://i.pinimg.com/originals/83/f8/03/83f8035b52d212cbd622930191f28ae8.jpg"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandKayozz.js]: again, different lines

File: models/Discord/Commands/DiscordCommandKtlwjec.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
...llbande.at/fellmiautschn/Nightwish11/Dr.Katz.gif", "https://cdn.discordapp.com/attachments/357600517310119937/357600564571537408/553d39b9789fac50540bd9fabd9df7db.jpg", "https://cdn.discordapp.com/attachments/357600517310119937/357600759203758101/40c5bumqbufy.jpg", "https://thumbs.dreamstime.com/z/doctor-cat-24832703.jpg", "https://imgs.tuts.dragoart.com/how-to-draw-doctor-cat-doctor-cat_1_000000010310_5.jpg", "https://d2gg9evh47fn9z.cloudfront.net/800px_COLOURBOX17208252.jpg", "http://images.bigcartel.com/product_images/55001423/dcplush.jpg?auto=format&fit=max&h=1000&w=1000", "https://img3.stockfresh.com/files/c/cynoclub/m/85/1542883_stock-photo-doctor-cat.jpg", "https://cdn.shopify.com/s/files/1/1312/6455/products/doctor_cat_480x480.PNG?v=1471114517", "http://l7.alamy.com/zooms/9a630ea301be4b6b804cb96665352181/black-cat-with-a-habit-of-doctor-in-front-of-white-background-cffgp3.jpg", "https://d3v8fhblas9eb9.cloudfront.net/i/wp-content/uploads/2013/08/CURIOUS-INTERESTED-CAT-DOCTOR.jpg", "https://cdn.discordapp.com/attachments/357600517310119937/357600759711531019/katie_and_orange_nurse2.jpg", "https://cdn.discordapp.com/attachments/357600517310119937/357600759711531018/drcat.PNG"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandKtlwjec.js]: See above

File: models/Discord/Commands/DiscordCommandManatee.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandManatee extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("manatee", "Pictures of very menacing manatees", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["https://cdn.discordapp.com/attachments/134722036353204224/357594548039581697/4961533005dc5510e49af2ea3789c80a--manatee-drawing-manatee-painting.jpg", "https://i.imgur.com/bCguwdn.jpg", "https://i.imgur.com/Z6IF4gR.jpg", "https://i.imgur.com/5VIfWhF.png", "https://i.imgur.com/gXYhvki.jpg", "https://i.imgur.com/cSXO2i4.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357626080414400512/unknown.png", "https://i.imgur.com/Wmrqu0U.jpg", "https://i.imgur.com/9d8xCRQ.jpg", "https://i.imgur.com/OlBm7g2.png", "https://i.imgur.com/xwEhrno.jpg", "http://i.imgur.com/jexXwfD.jpg?1"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandManatee.js]: Put them into seperate lines. Also, using discordapp links is bad, as some networks (most notably my campus') refuse to load images from there. I suggest hosting them on imgur or something instead

File: models/Discord/Commands/DiscordCommandOak.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandOak extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("oak", "Pigdures of counzil blease :DDDD", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["https://cdn.discordapp.com/attachments/134722036353204224/357308710315622400/75effd022f2f3beed48fa58027741ac1c76b112ed184bd1c37be93bc9786dd5a.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357308256932200468/1502487435509.png", "https://cdn.discordapp.com/attachments/134722036353204224/357215330742960138/1478923206629.png", "https://cdn.discordapp.com/attachments/134722036353204224/355046168851382275/1490619210410.png", "https://memestatic.fjcdn.com/pictures/Gondola_1ea063_6170689.jpg", "https://memes4.fjcdn.com/pictures/Gondola_b416c4_6170689.jpg", "https://memes4.fjcdn.com/pictures/Gondola_c27111_6170689.jpg", "https://memestatic.fjcdn.com/pictures/Gondola+art_a1ac9a_6170689.jpg", "https://memes4.fjcdn.com/pictures/Gondola_730f7e_6170689.jpg", "http://i.imgur.com/SVwB908.png", "https://cdn.discordapp.com/attachments/134722036353204224/355046544652894208/1481840106044.png"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandOak.js]: multiple lines

File: models/Discord/Commands/DiscordCommandPartheo.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandPartheo extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("partheo", "Pictures of an arsonist", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["http://app.pomaki.com/wp-content/uploads/2016/06/sprite-soda-can.jpg", "https://i.imgur.com/sGIuy8c.jpg", "https://www.sprite.com/content/dam/sprite2016/sprite_logo_green2.png", "https://i.imgur.com/hQx6shr.png", "https://i.imgur.com/gYdx4Mu.png", "https://d1l5iy72tii4pp.cloudfront.net/wp-content/uploads/2016/11/Firefighter-Salary.jpg", "https://i.imgur.com/CnaXPZV.png", "https://i.imgur.com/mlv9srS.jpg", "https://i.imgur.com/O9ixYcd.jpg", "https://i.imgur.com/D3CV2jw.jpg", "http://25.media.tumblr.com/tumblr_m6azl24fiI1qmt0h8o1_500.jpg", "https://i.redd.it/l2xun0i7tvqy.png"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandPartheo.js]: MuLtIpLe LiNeS

File: models/Discord/Commands/DiscordCommandSimon.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+const DiscordCommand = require('../DiscordCommand.js');
+
+class DiscordCommandSimon extends DiscordCommand {
+
+  constructor(subsystem) {
+    super("simon", "Pictures of the Dæne", undefined, subsystem, true);
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["https://cdn.discordapp.com/attachments/134722036353204224/357596897050886144/download.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357598415711895552/bwjftcz5ingfji28la3h.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/349623769779666946/download.jpg", "https://i.imgur.com/uDsHVLW.jpg", "https://i.imgur.com/1X91zDV.jpg", "https://i.imgur.com/TIzPllv.jpg", "https://i.imgur.com/MF5a1I8.png"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandSimon.js]: mUlTiBlE lInEs

File: models/Discord/Commands/DiscordCommandTrump.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+  }
+
+  onRun(message, permissions, args) {
+    var responses = ["https://cdn.discordapp.com/attachments/134722036353204224/357313396645036033/image.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357307859412975628/1504461825252.png", "https://cdn.discordapp.com/attachments/134722036353204224/357306304030703627/image.jpg", "http://i0.kym-cdn.com/photos/images/original/001/213/930/813.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357306988495110144/FB_IMG_1500982621533.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357306803979026433/1475092477289.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357306962444025859/1475091279184.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357307959295868928/1475090924184.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357301669312331788/image.jpg", "https://cdn.discordapp.com/attachments/134722036353204224/357278447216754689/1478453994669.png", "https://cdn.discordapp.com/attachments/134722036353204224/357278183013482496/image.jpg", "http://i.imgur.com/MQinI89.jpg", "http://i.imgur.com/FJMoq.jpg"];
```
Comment (nichlas0010) [inline models/Discord/Commands/DiscordCommandTrump.js]: again, multiple lines

**Comment by ncaaseawolf** [comment]
there is no !hitler that shows a picture of me
oh wait
im not staff
ignore me

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Where is  !1fbff5f83b23d39d38b1dfcb4cac8d9b

**Comment by nichlas0010** [comment]
👍

**Pull request merged**